### PR TITLE
Return bug fix

### LINKED ISSUE #266: Bug where global public variables that interact with the secret are always treated as return values in the contract (even if they are not returned).
Test on Assign-Return, Assign-Return1 and Booleans (with returns added). Also test BucketofBalls which also seems to have a EVM transaction reverted bug caused by the returns.

### PR DESCRIPTION
**Pull request by SwatiEY**
closes #266

### CODE REVIEW

File: src/boilerplate/contract/solidity/nodes/ContractBoilerplateGenerator.ts [hunk 1]
```diff
@@ -118,7 +118,7 @@ class ContractBoilerplateGenerator {
       let paramtype: string;
       let params : any[];
       let functionName: string;
-      
+      console.log('circuit', circuitParams);
```
Comment (lydiagarms) [inline src/boilerplate/contract/solidity/nodes/ContractBoilerplateGenerator.ts:121]: This this is for debugging?

File: src/boilerplate/contract/solidity/raw/ContractBoilerplateGenerator.ts [hunk 2]
```diff
@@ -149,6 +149,7 @@ class ContractBoilerplateGenerator {
         }
       }
         else if(type  === 'returnParameters') {
+          console.log('Input:', input);
```
Comment (lydiagarms) [inline src/boilerplate/contract/solidity/raw/ContractBoilerplateGenerator.ts:152]: Think this is also for debugging?

File: src/transformers/visitors/toContractVisitor.ts
Comment (lydiagarms) [inline src/transformers/visitors/toContractVisitor.ts:1]: There is a console.log to remove.

File: src/boilerplate/contract/solidity/nodes/ContractBoilerplateGenerator.ts (hunk 1 above)
Comment (SwatiEY) [reply src/boilerplate/contract/solidity/nodes/ContractBoilerplateGenerator.ts:121]: yes thanks.. I will remove all the consoles

File: src/boilerplate/contract/solidity/raw/ContractBoilerplateGenerator.ts (hunk 2 above)
Comment (SwatiEY) [reply src/boilerplate/contract/solidity/raw/ContractBoilerplateGenerator.ts:152]: Removed

File: src/transformers/visitors/toContractVisitor.ts
Comment (SwatiEY) [reply src/transformers/visitors/toContractVisitor.ts:1]: Removed

**Review by lydiagarms — APPROVED** [review]

**Pull request merged**